### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by hinoyso**
Refactoring and enhancing codebase based on mentor feedback to improve readability, efficiency, and maintainability.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const totalSizeLimit = 15;
```
Comment (danielmargolin) [inline script.js]: What is 15? 15 what? MB? KB? GB?
Consider switching to bytes and converting to MB only when you update the DOM

File: script.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  mbLeft.textContent = remainingStorage.toFixed(2) + ' MB LEFT';
+};
+
+const updateUploadedStorage = () => {
+  const usedStorageElement = document.getElementById('usedStorage');
+  const uploadedSizeInMB = totalUploadedSize / MB;
+  usedStorageElement.textContent = uploadedSizeInMB.toFixed(2) + ' MB';
+};
+
+const calculateRemainingStorage = () => totalSizeLimit - (totalUploadedSize / MB);
```
Comment (danielmargolin) [inline script.js]: If you don't read from localStorage, why do you need this logic? `totalUploadedSize` will always be 0, no?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  const validFileExtensionsRegex = /\.(gif|jpe?g|tiff?|png|webp|bmp)$/i;
+  const remainingStorage = calculateRemainingStorage();
+
+  selectedFiles.forEach((selectedFile) => {
+    const fileName = selectedFile.name;
+
+    if (validFileExtensionsRegex.test(fileName)) {
+      const fileSize = selectedFile.size;
+
+      if (fileSize / MB > remainingStorage) {
```
Comment (danielmargolin) [inline script.js]: Consider switching to positive approach:
```
if (fileSize / MB < remainingStorage) {
      totalUploadedSize += fileSize;
        
      } else {
alert("There is not enough space on the disk for file: " + fileName);
}

```

**Comment by danielmargolin** [comment]
Overall looks great!
Left some small comments
Please fix them before merging

**Review by danielmargolin — CHANGES_REQUESTED** [review]

**Review by danielmargolin — APPROVED** [review]

**Comment by hinoyso** [comment]
Hi Daniel,

I've made the necessary changes to address the issues you pointed out. The code has been updated according to your feedback.

Thank you for your valuable comments!

**Pull request merged**